blog/app/models/comment.rb

```diff
@@ -0,0 +1,3 @@
+class Comment < ActiveRecord::Base
+  belongs_to :article
```

Comment:
Although your application has no logic that would ever allow a comment to be created without an article you should still confirm that it is there.

Since if you delete an article the comment is also destroyed
```ruby
class Article < ActiveRecord::Base
	has_many :comments, dependent: :destroy
```

your article should "match"